PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1462,9 +1462,9 @@ Do you want to save your changes before switching themes?"/> <!-- HowToReproduce
             <ColumnVal name="Value"/>
             <ColumnHex name="Hex"/>
             <ColumnChar name="Character"/>
-            <ColumnHtmlNumber name="HTML Number"/>
-            <ColumnHtmlName name="HTML Code"/>
-            <ColumnHtmlHexNb name="HTML Hex Number"/>
+            <ColumnHtmlName name="HTML Name"/>
```

Comment:
I hesitated to ask renaming  "HTML Code" to "HTML Name", but I didn't, because personally I prefer the term "HTML code". Or it should be HTML Entity Name - too long to bad.

Reply:
@donho 
"HTML Name" is already used in `ansiCharPanel.cpp`
```
generic_string htmlNameStr = pNativeSpeaker->getAttrNameStr(L"HTML Name", "AsciiInsertion", "ColumnHtmlName");
```

Reply:
Yes, I know. That's why I said "I hesitated":
Even "HTML Name" is used as English column label, I thought "HTML Code" is more meaningful for translation.
But NM.